windows/web/slimevr_web_installer.nsi

```diff
@@ -27,7 +27,7 @@ InstallDir "$PROGRAMFILES\SlimeVR Server" ; $InstDir default value. Defaults to
 ShowInstDetails show
 ShowUninstDetails show
 
-BrandingText "SlimeVR Installer 0.1.7"
+BrandingText "SlimeVR Installer 0.1.8"
```

Comment:
Didn't we bump to v0.1.7 4 hours ago?

Reply:
yes, keep it 0.1.7 as it wasn't released yet

Reply:
ok